src/current/v22.2/changefeed-sinks.md

```diff
@@ -215,13 +215,17 @@ You can use [Google's Pub/Sub emulator](https://cloud.google.com/pubsub/docs/emu
 
 ### Pub/Sub topic naming
 
-When running a `CREATE CHANGEFEED` statement to Pub/Sub, it will try to create a topic automatically. When you do not specify the topic in the URI with the [`topic_name`](create-changefeed.html#topic-name-param) parameter, the changefeed will use the table name to create the topic name. If the topic already exists in your Pub/Sub sink, the changefeed will write to it. You can also use the [`full_table_name`](create-changefeed.html#full-table-option) option to create a topic using the fully qualified table name.
+When running a `CREATE CHANGEFEED` statement to a Pub/Sub sink, consider the following regarding topic names:
 
-The output from `CREATE CHANGEFEED` will display the job ID as well as the topic name(s) that the changefeed will emit to.
+- Changefeeds will try to create a topic automatically. When you do not specify the topic in the URI with the [`topic_name`](create-changefeed.html#topic-name-param) parameter, the changefeed will use the table name to create the topic name.
+- If the topic already exists in your Pub/Sub sink, the changefeed will write to it.
+- Changefeeds watching multiple tables will write to multiple topics corresponding to those table names.
+- The [`full_table_name`](create-changefeed.html#full-table-option) option will create a topic using the fully qualified table name for each table the changefeed is watching.
+- The output from `CREATE CHANGEFEED` will display the job ID as well as the topic name(s) that the changefeed will emit to.
```

Comment:
```suggestion
- The output from `CREATE CHANGEFEED` will display the job ID as well as the topic name(s) to which the changefeed will emit.
```

I've always found it weird to end sentences in prepositions.

Reply:
You either love 'em or you hate 'em 😁 

Though I'm fine either way, so I'll add your change here.